meta-lmp-base/recipes-bsp/efitools/efitools_git.bb

```diff
@@ -18,6 +18,8 @@ SRC_URI += " \
 
 # UnLock needs the user keys
 SRC_URI:append = "${@bb.utils.contains('UEFI_SIGN_ENABLE', '1', ' file://unlock.patch file://unlock.conf', '', d)}"
+# the above UEFI_SIGN_ENABLE makes this recipe machine specific
+PACKAGE_ARCH = "${MACHINE_ARCH}"
```

Comment:
maybe we can bring the SRC_URI:append to meta-lmp-bsp and make it all machine specific

Reply:
I understand our suggestion but this implies moving also the manipulation of the files in https://github.com/foundriesio/meta-lmp/commit/fcd7286c25c2f617144f3ce6f41bd04f09dbd767

Reply:
if the file are only present for when it's machine specific

but anything, only a suggestion, you can go ahead with it as is.